### ISSUE TITLE
Datapoint 9.002 with direct kelvin input.

### ISSUE THREAD

**Issue by Supergiovane** [Supergiovane/node-red-contrib-knx-ultimate#283]
That's very kind of you, but I had some spare time tonight and had another look at the figures I put into the Excel sheet.

It seems to be pretty simple: 1000000 / Mirek value = Kelvin-value
That's no scientific formula I guess, but it's pretty precise within the usual range and is very similar to the values from ISE HUE.

I can use a simple function node to rescale your values to Mirek and convert to/from Kelvin, so it does not need to be done inside the node. If you like the idea you can put it there of course as DPT 9.002 

(Btw. I never found a satisfying conversion from RGB to color temperature or vice versa.)

_Originally posted by @SmartyKNX in https://github.com/Supergiovane/node-red-contrib-knx-ultimate/issues/281#issuecomment-1793862555_

**Comment by SmartyKNX** [comment]
I tried the new version last weekend and the values are a little bit out of my expectations.

Using the HUE app and selecting the center of the total range results in values of 4300K.  I think this value is way to blueish for the actual impression from the HUE light.

Before you implemented this function, I took the 7.600 DPT with values from 0-65k and rescaled these values to the mirek-range from 153-500. 1000000/MirekValue results in pretty reasonable kelvin values. However these values are not the values I get, when using the direct output of your 9.002DPT.

Of course if you are happy with values keep them, because I can use my own conversion in Node Red anyway.
